WebfrontCore/Views/Client/Find/Index.cshtml

```diff
@@ -20,7 +20,11 @@
                         <color-code value="@client.Name" allow="@ViewBag.EnableColorCodes"></color-code>
                     </a>
                 </div>
-                <div class="col-4 level-color-@client.LevelInt">@client.Level</div>
+                @if (ViewBag.Authorized){
```

Comment:
Should be configuration driven.
Add a new bool property to ApplicationConfiguration.cs called `EnableAdminPrivacy` or something like that. Then set that value on the viewbag in `BaseController.cs`.
This statement will then be something like: `@if (!ViewBag.Authorized && ViewBag.EnableAdminPrivacy) { // show fake level } else { // show actual level }} `